DataFormats/PatCandidates/interface/IsolatedTrack.h

```diff
@@ -3,15 +3,21 @@
 
 /*
   \class    pat::IsolatedTrack IsolatedTrack.h "DataFormats/PatCandidates/interface/IsolatedTrack.h"
-  \brief Small class to store key info isolated packed cands    
-   pat::IsolatedTrack stores important info on isolated packed PF candidates,
-   as a convenience to analyses to prevent having to loop over all pfcands.
+  \brief Small class to store key info on isolated tracks
+   pat::IsolatedTrack stores important info on isolated tracks. Draws from
+   packedPFCandidates, lostTracks, and generalTracks.
   \author   Bennett Marsh
 */
 
 #include "DataFormats/Candidate/interface/LeafCandidate.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackBase.h"
+#include "DataFormats/TrackReco/interface/HitPattern.h"
 #include "DataFormats/PatCandidates/interface/PackedCandidate.h"
 #include "DataFormats/PatCandidates/interface/PFIsolation.h"
+#include "CondFormats/HcalObjects/interface/HcalChannelStatus.h"
+#include "CondFormats/EcalObjects/interface/EcalChannelStatusCode.h"
```

Comment:
I have just realized that this dependency is not legal
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts

For persistency, the status codes will need to be converted to plain uint16_t (ecal) and uint32_t (hcal).